app/vmctl/remoteread.go

```diff
@@ -46,19 +48,19 @@ func (rrp *remoteReadProcessor) run(ctx context.Context, silent, verbose bool) e
 
 	question := fmt.Sprintf("Selected time range %q - %q will be split into %d ranges according to %q step. Continue?",
 		rrp.filter.timeStart.String(), rrp.filter.timeEnd.String(), len(ranges), rrp.filter.chunk)
-	if !silent && !prompt(question) {
+	if !rrp.isSilent && !prompt(question) {
 		return nil
 	}
 
 	var bar *pb.ProgressBar
-	if !silent {
+	if !rrp.isSilent {
 		bar = barpool.AddWithTemplate(fmt.Sprintf(barTpl, "Processing ranges"), len(ranges))
 		if err := barpool.Start(); err != nil {
```

Comment:
Also, it doesn't fix any how an issue with stats -`Print out stats for migration every N interval.`

 Promising solution to me - tweak barpool methods. In case when TTY is not avaible. barpool.Increment method must print with `log.Print()` counter every N-minute. It allows to hide TTY related code from processors and greatly simplify code. Since you don't need to check `if !rrp.isSilent {`.

Reply:
I have updated the logic. How it works now:
1. `gloablSilence` or `s` flag will disable prompt and bar logs. Users will see only logs which not disabled by silence flags.
2. In other cases progress will be printed. If we have tty progress will be in one line. If tty doesn't exist the progress will be incrementaly prints line by line.
```Continue import process with filter 
        filter: match[]={__name__!=""}
        start: 2023-05-28T00:00:00Z
        end: 2023-05-29T00:00:00Z:
Total: 64.00 KiB ↖ Speed: ? p/s 
Total: 384.00 KiB ↗ Speed: ? p/s 
Total: 832.00 KiB ↘ Speed: 1.35 MiB p/s 
Total: 2.71 MiB ↙ Speed: 1.35 MiB p/s 
Total: 5.89 MiB ↖ Speed: 1.35 MiB p/s 
Total: 7.30 MiB ↗ Speed: 1.96 MiB p/s 
Total: 10.14 MiB ↘ Speed: 1.96 MiB p/s ```